main.tf

```diff
@@ -30,8 +30,8 @@ resource "aws_security_group_rule" "http_ingress" {
   from_port         = "${var.http_port}"
   to_port           = "${var.http_port}"
   protocol          = "tcp"
-  cidr_blocks       = ["${var.http_ingress_cidr_blocks}"]
-  prefix_list_ids   = ["${var.http_ingress_prefix_list_ids}"]
+  cidr_blocks       = "${var.http_ingress_cidr_blocks}"
```

Comment:
was this tested?
TF 0.11 had issues when you don't specify `[]` for lists even if the provided var is a list itself